solutions/usaco-945.mdx

```diff
@@ -0,0 +1,73 @@
+---
+id: usaco-945
+source: USACO Gold 2019 US Open
+title: Snakes
+author: Sofia Yang
+---
+
+[Official Analysis and Code(C++)](http://www.usaco.org/current/data/sol_snakes_gold_open19.html)
```

Comment:
```suggestion
[Official Analysis and Code (C++)](http://www.usaco.org/current/data/sol_snakes_gold_open19.html)
```

Reply:
waait isn't is supposed to just be `[Official Analysis (C++)]`